pkg/bootstrapper/bootstrapper.go

```diff
@@ -558,7 +564,21 @@ func (wmcb *winNodeBootstrapper) generateInitialKubeletArgs(args map[string]stri
 	if wmcb.nodeIP != "" {
 		kubeletArgs = append(kubeletArgs, "--node-ip="+wmcb.nodeIP)
 	}
-	return kubeletArgs
+
+	var hostname string
+	var err error
+	switch wmcb.platformType {
+	case "AWS":
```

Comment:
lets make this  `aws` instead of `AWS`, as this is a flag being passed in, and all lowercase is more commonly used

Reply:
This could potentially be made a constant in the new package you will introduce.